integration-tests/olminstall/helpers/patch_cluster_pull_secret.py

```
@@ -0,0 +1,151 @@
+import base64
+import json
+import subprocess
+import sys
+from pathlib import Path
+from typing import Any
+
+QUAY_SECRET_PATH = Path("/var/secret/quay/.dockerconfigjson")
+
+
+def run_oc(args: list[str], *, check: bool = True) -> subprocess.CompletedProcess[str]:
+    return subprocess.run(["oc", *args], check=check, capture_output=True, text=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)